src/vars.js

```diff
@@ -129,26 +147,32 @@ exports.browser = {
   HTMLIFrameElement    : false,
   HTMLImageElement     : false,
   HTMLInputElement     : false,
-  HTMLIsIndexElement   : false,
```

Comment:
This removal may reflect the state of modern browsers, but many of JSHint's consumers target legacy environments. That means this particular change constitutes a breaking change, so we'll have to defer it for the next major release. Could you instead add an in-line comment with the research you reported in the pull request description?

Reply:
@jugglinmike I modified the pull request title and description. I hope those changes suffice.

Support for `isindex` in the most used browser—Google Chrome—was apparently pulled three years and nine months ago, so this seems like a low-risk change.

Reply:
@jugglinmike I've changed the WHATWG HTML GitHub issue reference in the description from 1088 ("Remove `isindex` parsing special case") to 1095 ("Remove `<isindex>` and `<input name=isindex>`"). The latter more directly addresses the removal of `HTMLIsIndexElement`.

Reply:
That's more great research, thanks! Though it's good evidence for the low risk of removal, I still don't consider the benefit to be great enough to warrant it. Would you mind adding this back in with a comment describing your findings?

Reply:
@jugglinmike Yes. I:

* restored `HTMLIsIndexElement` with an ECMAScript comment preceding it.
* updated the description with `del` elements.
* updated the pull request title.